### PR TITLE
Day thirty two

### PR DESCRIPTION
**Pull request by Qairkhan**
Errors were updated when updating the profile. New profile descriptions added. * Note: I18N will be updated.

### CODE REVIEW

File: src/api/api.js [hunk 1]
```diff
@@ -50,6 +50,11 @@ const usersAPI = {
       },
     });
   },
+
+  saveProfile: (profile) => {
```
Comment (Dmitry-White) [inline src/api/api.js]: Don't forget about arrow functions short form:
`() => { return ... }` might be written shorter `() => ...`
Arrow functions in the short form without {} will immediately return what's written after =>.
`saveProfile: (profile) => instance.put(`${ENDPOINTS.SAVE_PROFILE}`, profile),` looks much better.

File: src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import {
+  createField,
+  Input,
+  TextArea,
+} from "../../common/FormsControls/FormsControls";
+import { reduxForm } from "redux-form";
```
Comment (Dmitry-White) [inline src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx]: Consider fixing imports ordering and grouping.

File: src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          [],
+          TextArea
+        )}
+      </div>
+      <div>
+        <b>About me:</b> {createField("aboutMe", "aboutMe", [], TextArea)}
+      </div>
+      <div>
+        <b>Contacts:</b>{" "}
+        {Object.keys(profile.contacts).map((key) => {
```
Comment (Dmitry-White) [inline src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx]: Consider breaking this line down into several functions/variables.
E.g. `Object.keys(...).map( () => {...})` could be
`const arr = Object.keys(...);`
`const mapHandler = () => {...};`
`const result = arr.map(mapHandler);`

File: src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import {
+  createField,
+  Input,
+  TextArea,
+} from "../../common/FormsControls/FormsControls";
+import { reduxForm } from "redux-form";
+
+const ProfileDescriptionForm = ({handleSubmit, profile }) => {
+  return (
+    <form onSubmit={handleSubmit}>
```
Comment (Dmitry-White) [inline src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx:49]: Consider breaking down this JSX into several readable functions/variables.

File: src/components/common/FormsControls/FormsControls.js [hunk 5]
```diff
@@ -1,11 +1,62 @@
 import React from "react";
+import { Field } from "redux-form";
 
-const TextareaCastom = ({input, meta, ...props}) => {
+const TextareaCastom = ({ input, meta, ...props }) => {
```
Comment (Dmitry-White) [inline src/components/common/FormsControls/FormsControls.js]: There's a typo in the name.

**Review by Dmitry-White — CHANGES_REQUESTED** [review]

File: src/api/api.js (hunk 1 above)
Comment (Qairkhan) [reply src/api/api.js]: done

File: src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx (hunk 3 above)
Comment (Qairkhan) [reply src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx]: done

File: src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx (hunk 2 above)
Comment (Qairkhan) [reply src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx]: done

File: src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx (hunk 4 above)
Comment (Qairkhan) [reply src/components/Profile/ProfileInfo/ProfileDescriptionForm.jsx:49]: done

File: src/components/common/FormsControls/FormsControls.js (hunk 5 above)
Comment (Qairkhan) [reply src/components/common/FormsControls/FormsControls.js]: done

**Review by Dmitry-White — APPROVED** [review]

**Pull request merged**